Revit_oM/Requests/FilterByWorkset.cs

```diff
@@ -23,10 +23,10 @@
 using BH.oM.Data.Requests;
 using System.ComponentModel;
 
-namespace BH.oM.Adapters.Revit
+namespace BH.oM.Adapters.Revit.Requests
 {
-    [Description("IRequest that filters all elements in a given workset.")]
-    public class WorksetRequest : IRequest
+    [Description("IRequest that filters all elements in a given Revit workset.")]
+    public class FilterByWorkset : IRequest
```

Comment:
Same observation https://github.com/BHoM/Revit_Toolkit/pull/609/files#r395599880? I see you have done it on purpose here. I won't enforce compliance, but maybe it's worth having other people opinion on this?

Reply:
As above, with the number of >20 the `Request` suffix started to be a burden that we decided to drop.

In general, I have no strong opinion about the naming as long as it is readable to anyone else than the originator and his/her fellow developers (which I noticed happens to be the case). So, compliance yes, but not at the cost of the UX - maybe a good trade off would be to rename the classes to be compliant and add human readable Create method, same as done with `FilterRequest` and `SelectionRequest`?